WORKSPACE

```diff
@@ -69,14 +69,3 @@ rules_proto_toolchains()
 load("@com_github_bazelbuild_buildtools//buildifier:deps.bzl", "buildifier_dependencies")
 
 buildifier_dependencies()
-
-# We don't use any nodejs but this includes a rule for publishing releases to npm
-http_archive(
-    name = "build_bazel_rules_nodejs",
-    sha256 = "b6670f9f43faa66e3009488bbd909bc7bc46a5a9661a33f6bc578068d1837f37",
-    urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/1.3.0/rules_nodejs-1.3.0.tar.gz"],
-)
-
-load("@build_bazel_rules_nodejs//:index.bzl", "node_repositories")
```

Comment:
I assume this and the next line should be kept, otherwise

Reply:
Whoops, you're absolutely right. `dG` at it's finest :)